vkHW/vkHW/Views/Transitions/CustomPopAnimator.swift

```diff
@@ -0,0 +1,63 @@
+// CustomPopAnimator.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Анимации попа
```

Comment:
```suggestion
/// Анимации попа
```
что за поп такой, аниматорщик)